modules/osx/init.zsh

```diff
@@ -19,3 +19,13 @@ alias cdf='cd "$(pfd)"'
 
 # Pushes directory to the current Finder directory.
 alias pushdf='pushd "$(pfd)"'
+
+# Locks screen.
+alias lock='/System/Library/CoreServices/Menu\ Extras/User.menu/Contents/Resources/CGSession -suspend'
```

Comment:
Unfortunately just doing the `pmset` command by itself is not enough to actually lock the screen, it will only put the display to sleep. Looking at [a question on AskDifferent](http://apple.stackexchange.com/q/80058/55251) it seems the lock command @musophob wrote is the best option to lock the screen regardless of user settings.